tomcat/src/main/java/org/apache/coyote/http11/request/RequestMethod.java

```diff
@@ -0,0 +1,30 @@
+package org.apache.coyote.http11.request;
+
+import org.apache.coyote.http11.exception.InvalidRequestException;
+
+import java.util.Arrays;
+
+public enum RequestMethod {
+    GET,
+    POST,
+    PATCH,
+    UPDATE,
+    DELETE,
+    ;
+
+    RequestMethod() {
+    }
+
+    public static RequestMethod getRequestMethod(String value) {
+        return Arrays.stream(RequestMethod.values())
+                .filter(requestMethod -> requestMethod.name().equals(value))
+                .findFirst()
+                .orElseThrow(InvalidRequestException::new);
+    }
+
+    public boolean isSameRequestMethod(String value) {
+        return Arrays.stream(RequestMethod.values())
+                .anyMatch(requestMethod -> requestMethod.name().equals(value));
```

Comment:
`requestLine.getRequestMethod().isSameRequestMethod("GET")` 이걸 통해서 GET 메소드인지 확인하려고 하는것 같아요!
다만 지금과 같은 로직으로 실행되면 무조건 true가 나오는 상황이 발생하여 리다이렉트가 불가합니다!

위의 로직 대신 이건 어떨까요 ?
```suggestion
                return this.name().equals(value);
```